modules/integration_aws-elasticsearch/conf/09-master-cpu.yaml

```diff
@@ -0,0 +1,19 @@
+module: AWS Elasticsearch
+name: "Master CPU"
+
+transformation: ".min(over='20m')"
+aggregation: ""
+filtering: "filter('namespace', 'AWS/ES') and filter('stat', 'upper') and filter('NodeId', '*')"
+
+signals:
+  signal:
+    metric: MasterCPUUtilization
+
+rules:
+  major:
+    threshold: 50
```

Comment:
AWS [recommends](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/managedomains-cloudwatchmetrics.html#managedomains-cloudwatchmetrics-master-node-metrics) _increasing the size of the instance type when this metric reaches 60 percent_.

Reply:
Done